packages/vitest/src/node/cli/cli-api.ts

```diff
@@ -184,6 +189,48 @@ export function processCollected(ctx: Vitest, files: File[], options: CliOptions
   return formatCollectedAsString(files).forEach(test => console.log(test))
 }
 
+export function outputFileList(files: WorkspaceSpec[], options: CliOptions) {
+  if (typeof options.json !== 'undefined') {
+    return outputJsonFileList(files, options)
+  }
+
+  return formatFilesAsString(files, options).map(file => console.log(file))
+}
+
+function outputJsonFileList(files: WorkspaceSpec[], options: CliOptions) {
+  if (typeof options.json === 'boolean') {
+    return console.log(JSON.stringify(formatFilesAsJSON(files), null, 2))
+  }
+  if (typeof options.json === 'string') {
+    const jsonPath = resolve(options.root || process.cwd(), options.json)
```

Comment:
```suggestion
    const jsonPath = relative(options.root || process.cwd(), options.json)
```

should be imported from `pathe`, can't add the comment there unfortunetly

Reply:
did it in 99afa71